toolbox/Dockerfile

```diff
@@ -15,7 +15,6 @@
 FROM ubuntu:xenial
 MAINTAINER Jared Watts <jared.watts@quantum.com>
 
-
 RUN apt-get update && \
```

Comment:
I don't know of any problems using the jewel clients with a kraken cluster.  Sounds like it should be fine.  It just _feels_ safer to match up client/server versions.